### PR TITLE
add image and input

### PR DESCRIPTION
**Pull request by Divya70**
add different style of image
1.  responsive image
2. rounded image
imput form
radio button and checkbox

res image
rounded image
input

### CODE REVIEW

**Review by sagar455 — APPROVED** [review]
Looks good

File: pages/Form/form.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+.input-iframe{
+      width: 1000px; 
+    height: 500px;
+    border:0;
+    margin-top: 2rem;
+}
+
+/* radio button code */
+
+.button-cont{
```
Comment (Ayushi20-19) [inline pages/Form/form.css:69]: use prettier that will improve formatting

**Review by Ayushi20-19 — APPROVED** [review]
write only necessary code of a particular component in its CSS file and make a separate file for other code.

File: pages/Form/form.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      padding: 0.5rem;
+      font-size: 1.3rem;
+      
+}
+.error-message{
+      margin-left: 2rem;
+      color: red;
+}
+.input-iframe{
+      width: 1000px;
```
Comment (Ayushi20-19) [inline pages/Form/form.css:61]: why use px? u can use rem here also

**Pull request merged**